### PR TITLE
Add option to split multiple terms into AND list

### PR DESCRIPTION
**Pull request by adamdickmeiss**
CQLParser.andMultipleTerms = true
makes the CQL parser produce AND nodes for multiple terms
rather than concatenating them.

### CODE REVIEW

File: src/main/java/org/z3950/zing/cql/CQLParser.java [hunk 1]
```diff
@@ -260,7 +265,18 @@ private CQLNode parseTerm(String index, CQLRelation relation)
 	    debug("index='" + index + ", " +
 		  "relation='" + relation.toCQL() + "'");
 	}
-	CQLTermNode node = new CQLTermNode(index, relation, all.toString());
+        CQLNode node = null;
+        if (this.andMultipleTerms && "=".equals(relstr)) {
```
Comment (julianladisch) [inline src/main/java/org/z3950/zing/cql/CQLParser.java:269]: For the "all" relation this is a correct expansion.
For the "==", the "adj" and the "any" relation this is wrong.

http://www.loc.gov/standards/sru/cql/contextSets/theCqlContextSet.html says:
> =
> This is the default relation, and the server can choose any appropriate relation
> For a string term, either 'adj' or '==' as appropriate for the index and term.

Comment (adamdickmeiss) [reply src/main/java/org/z3950/zing/cql/CQLParser.java:269]: The standard does not apply here.. The standard does not allow multiple terms after a field..

Such as title=a b
The standard BNF actually only allows title="a b".

And thx for looking into this.

So we are in any case describing something that isn't standard. Believe me I was part of the team (Wrote the grammar)

**Comment by julianladisch** [comment]
This Jira for this pull request https://issues.folio.org/browse/RMB-428 explains why it has been closed as "Won't do".

**Comment by adamdickmeiss** [comment]
Yes. And here we are 14 months later and it is not possible to express phrase search in the UI

**Comment by MikeTaylor** [comment]
Can we not do phrase search in the UI  just be enclosing in quotes?

**Comment by adamdickmeiss** [comment]
Don't think so, Mike. Better ask that on Slack.

**Pull request closed without merging**